website/content/maintainers/community/vendor-neutrality.md

```diff
@@ -0,0 +1,84 @@
+
+This document provides guidelines on what it means for a project within the CNCF to be vendor-neutral. While there are many aspects of vendor neutrality, this document specifically dives into what it means for a project to communicate, host, drive architecture decisions, and govern in a vendor neutral way.
+
+## Communication
+
+Projects manage their own communications channels like websites, blogs, social media accounts, and Slack. All of these are great ways for [non-service providing adopters](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter) to keep up to date on the project and learn about how the project can help solve their business challenges.
+
+Many adopters may be interested in commercial support for CNCF projects, but may not be aware of all of the commercial offerings from different companies in the ecosystem that leverage the project. They may turn to the project for more information on how to gain commercial support and/or software for their needs. For projects, the key principle is that one company should not be favored over any other companies offering the same services. This means that each vendor should have fair representation and objective information should be provided by the project. Some general guidelines:
+
+1. Websites
+    1. Please refer to the CNCF [website guidelines](https://github.com/cncf/foundation/blob/main/website-guidelines.md)
+2. Blogs
+    2. Blogs may mention a vendor’s name as it relates to specific open source project, project deployments, adoption paths, their hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, but it shouldn’t feel like an advertisement for the services or company
+    3. Blogs hosted on project websites should be about the open source project, any references to enterprise offerings should point to the enterprise page of the project that features all vendors equally rather than linking to one specific vendor
+    4. Project blogs should not favor one vendor’s solution over another
+    5. Project blogs should keep other CNCF projects in a positive light
+3. Social media
+    6. Projects can choose whether they want to repost content from vendors, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
+    7. Post/re-posts from project handles should not have content that disparages different vendors or CNCF projects
+4. Messaging platforms, such as Slack
+    8. It is up to the project whether vendors can post in their messaging channels, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
+    9. Vendors should avoid channel spamming or solicitation through private messages
```

Comment:
```suggestion
### Websites

*Please refer to the CNCF [website guidelines](https://github.com/cncf/foundation/blob/main/website-guidelines.md)

### Blogs

* Blogs may mention a vendor’s name as it relates to specific open source project, project deployments, adoption paths, their hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, but it shouldn’t feel like an advertisement for the services or company
* Blogs hosted on project websites should be about the open source project, any references to enterprise offerings should point to the enterprise page of the project that features all vendors equally rather than linking to one specific vendor
* Project blogs should not favor one vendor’s solution over another
* Project blogs should keep other CNCF projects in a positive light

### Social media

* Projects can choose whether they want to repost content from vendors, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
* Post/re-posts from project handles should not have content that disparages different vendors or CNCF projects

### Messaging platforms, such as Slack

* It is up to the project whether vendors can post in their messaging channels, however the rules of engagement should be consistent across vendors and ideally written down so they are clear to everyone
* Vendors should avoid channel spamming or solicitation through private messages
```